java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1878,6 +2083,48 @@ public Table stringSplit() {
     }
   }
 
+  /**
+   * Returns a column of lists of strings by splitting each string using whitespace as the delimiter.
+   */
+  public ColumnVector stringSplitRecord() {
+    return stringSplitRecord(-1);
+  }
+
+  /**
+   * Returns a column of lists of strings by splitting each string using whitespace as the delimiter.
+   * @param maxSplit the maximum number of records to split, or -1 for all of them.
+   */
+  public ColumnVector stringSplitRecord(int maxSplit) {
+    try (Scalar emptyString = Scalar.fromString("")) {
+      return stringSplitRecord(emptyString, maxSplit);
+    }
+  }
+
+  /**
+   * Returns a column of lists of strings by splitting each string using whitespace as the delimiter.
+   * @param delimiter UTF-8 encoded string identifying the split points in each string.
+   *                  Default of empty string indicates split on whitespace.
```

Comment:
Not sure "Default" is appropriate here since it is a required argument.  Also is the delimiter a sequence of single-character delimiters, any of which could be a split point in the string, or is there only one delimiter and it can be multi-character?  Would be good to clarify in the comment.